src/oc_erchef/apps/chef_index/src/chef_index_batch.erl

```diff
@@ -144,11 +143,11 @@ flush(State = #chef_idx_batch_state{item_queue = Queue,
     {PidsToReply, Timestamps, DocsToAdd} = lists:unzip3(Queue),
     Doc = wrap(DocsToAdd, State),
     Self = self(),
-    spawn_link(
+    spawn(
```

Comment:
It looks like we were  never doing anything with the termination notification, that's why the change to `spawn`?